CI: Submit the Application Payload to Microsoft store into CI

starnight opened the pull request:
Bring the actions to submit Endless Key to Microsoft store
automatically when release.

https://phabricator.endlessm.com/T33436

starnight committed:
CI: Submit the Application Payload to Microsoft store into CI

Bring the actions to submit Endless Key to Microsoft store
automatically when release.

https://phabricator.endlessm.com/T33436

danigm reviewed (approved):
Looks good to me

jprvita commented on .github/workflows/main.yml:
```diff
@@ -157,3 +157,62 @@ jobs:
+            -ReplacePackages `
+            -AutoCommit `
+            -Force `
+            -Verbose
```
I think we should add `-UpdateListings` here to also update the screenshots -- rationale in https://phabricator.endlessm.com/T33436#940050

starnight replied:
Updated & place some discussion in https://phabricator.endlessm.com/T33436#940115

jprvita replied:
Further discussion on the ticket.

starnight replied:
Updated!

jprvita reviewed (changes requested):
Further discussion on https://phabricator.endlessm.com/T33436#940316

starnight committed:
CI: Drop the screenshots and create dummies for Application Payload creation

To avoid confusion of having the screenshots in multiple places, drop
those in the repository and only maintain them on Microsoft store. Just
create dummy screenshots by parsing the PDP.xml downloaded from
Microsoft store in CI for the Application Payload creation.

https://phabricator.endlessm.com/T33436
